pkg/sql/opt/norm/testdata/rules/project_set

```diff
@@ -309,3 +309,120 @@ project
       │    └── columns: j:3
       └── zip
            └── json_array_elements(j:3) [outer=(3), immutable]
+
+# --------------------------------------------------
+# ConvertProjectionSetRawScanToKvScan
+# --------------------------------------------------
+
+# Basic case with constant values.
+norm expect=ConvertProjectionSetRawScanToKvScan
+SELECT * FROM crdb_internal.scan(crdb_internal.table_span(105))
```

Comment:
Perhaps this is obvious to people more familiar with SQL evaluation, but will something like `crdb_internal.table_span('users'::regclass::oid::int)` also work here?